### ISSUE TITLE
Node.js HTTP module is missing STATUS_CODES export

### ISSUE THREAD

**Issue by gajus** [facebook/flow#3688]
```js
import {
  STATUS_CODES
} from 'http';

```

is giving error:

```
test/helpers/createIncomingMessage.js:8
  8: import {STATUS_CODES} from 'http';
                   ^^^^^^^^^^^^ Named import from module `http`. This module has no named export called `STATUS_CODES`.
```

See https://nodejs.org/api/http.html#http_http_status_codes

**Comment by gajus** [comment]
... actually, it should be

```js
import {
  Server
} from 'http';

Server.STATUS_CODES;

```

but this gives an error too.

**Comment by nmote** [comment]
Fixed in #3617
